odl/contrib/datasets/images/examples/cambridge_images.py

```diff
@@ -0,0 +1,16 @@
+"""Example of the Cambridge images."""
+
+import odl.contrib.datasets.images as images
+
+images.brain_phantom(size=[200, 200]).show()
+images.resolution_phantom(size=[100, 123]).show()
+images.building().show()  # this is shown as gray scales ATM
+images.building(gray=True).show()
+images.rings(gray=True).show()
+images.rings(size=[100, 100]).show()  # this is shown as gray scales ATM
+images.blurring_kernel().show()
+
+# Run doctests
+# pylint: disable=wrong-import-position
+from odl.util.testutils import run_doctests
+run_doctests()
```

Comment:
This run_doctests in examples was a misstake by me in some other file and should not be there. Sorry about that :/